src/Sylius/Bundle/ShopBundle/Resources/views/Taxon/_horizontalMenu.html.twig

```diff
@@ -1,12 +1,15 @@
 {% macro item(taxon) %}
     {% import _self as macros %}
 
-    {% if taxon.children|length > 0 %}
+    {% set taxonChildren = taxon.children|filter(t => t.enabled == true) %}
```

Comment:
I suppose we should put this as the default filtration in `Taxon::getChildren()` method. It removes the business logic from twig (which is also 👍) and removes duplication (which is even more 👍, see next file). We could also introduce another method for taxon like `getAllChildren()` or something similar, if it's required somewhere (probably in the admin menu). Or the other way, we could have `getEnabledChildren()` method which would be used in shop (maybe it's even better choice) 🏇

Reply:
`enabledChildren` 👍 This way we'd reflect the `enabledVariants` introduced in toggleable product variant.